docs/src/ensemble_kalman_inversion.md

```diff
@@ -1,37 +1,58 @@
+Kalman inversion ([Iglesias et al, 2013](http://dx.doi.org/10.1088/0266-5611/29/4/045001)).
+The ensemble Kalman inversion (EKI) is a derivative-free ensemble optimization method that seeks
+to find the optimal parameters ``\theta \in \mathbb{R}^p`` in the inverse problem
 
-\\[ y = \mathcal{G}(\theta) + \eta, \\]
+```math
+y = \mathcal{G}(\theta) + \eta ,
+```
+
+where ``\mathcal{G}`` denotes the forward map, ``y \in \mathbb{R}^d`` is the vector of observations
+and ``\eta  \in \mathbb{R}^d`` is additive observational noise. Note that ``p`` is the
+size of the parameter vector ``\theta`` and ``d`` the size of the observation vector ``y``.
+
+Here, we take ``\eta \sim \mathcal{N}(0, \Gamma_y)`` from a ``d``-dimensional multivariate
+normal distribution with zero mean and covariance matrix ``\Gamma_y``.  This noise structure
+aims to represent the correlations between observations.
 
-where $\mathcal{G}$ denotes the forward map, $y \in \mathbb{R}^d$ is the vector of observations and $\eta \sim \mathcal{N}(0, \Gamma_y)$ is additive Gaussian observational noise. Note that $p$ is the size of the parameter vector $\theta$ and $d$ is taken to be the size of the observation vector $y$. The EKI update equation for parameter vector $\theta^{(j)}$ of ensemble member $j$ is
+The parameter vector of the ``j``-th ensemble member at the ``n``-th iteration is ``\theta^{(j)}_n``. The EKI update equation for parameter vector ``\theta^{(j)}`` is
 
 ```math
-\theta_{n+1}^{(j)} = \theta_{n}^{(j)} - \dfrac{\Delta t_n}{J}\sum_{k=1}^J\langle \mathcal{G}(\theta_n^{(k)}) - \bar{\mathcal{G}}_n, \Gamma_y^{-1}(\mathcal{G}(\theta_n^{(j)}) - y) \rangle \theta_{n}^{(k)},
+\theta_{n+1}^{(j)} = \theta_{n}^{(j)} - \dfrac{\Delta t_n}{J}\sum_{k=1}^J \left \langle \mathcal{G}(\theta_n^{(k)}) - \bar{\mathcal{G}}_n \, , \, \Gamma_y^{-1} \left ( \mathcal{G}(\theta_n^{(j)}) - y \right ) \right \rangle \theta_{n}^{(k)} ,
 ```
 
-where the subscript $n=1, \dots, N_{it}$ indicates the iteration, $J$ is the number of members in the ensemble and $\bar{\mathcal{G}}_n$ is the mean value of $\mathcal{G}(\theta)$ across ensemble members,
+where the subscript ``n=1, \dots, N_{it}`` indicates the iteration, ``J`` is the number of
+members in the ensemble, ``\bar{\mathcal{G}}_n`` is the mean value of ``\mathcal{G}(\theta_n)``
+across ensemble members,
 
 ```math
-\bar{\mathcal{G}}_n = \dfrac{1}{J}\sum_{k=1}^J\mathcal{G}(\theta_n^{(k)}).
+\bar{\mathcal{G}}_n = \dfrac{1}{J}\sum_{k=1}^J\mathcal{G}(\theta_n^{(k)}) ,
 ```
 
-The EKI algorithm is considered converged when the ensemble achieves sufficient consensus/collapse in parameter space. The final estimate $\bar{\theta}_{N_{it}}$ is taken to be the ensemble mean at the final iteration,
+and angle brackets denote the Euclidean inner product. By multiplying with ``\Gamma_y^{-1}``
+(which has dimensions ``y^2``) we render the inner product non-dimensional.
```

Comment:
I think you mean ``d \times d`` rather than ``y^2``

Reply:
I actually meant "units of y^2". Thanks.

Reply:
But I'll just remove the parenthetical remark.